javascript/modules/client/src/index.ts

```diff
@@ -1,8 +1,8 @@
 export * from "./client";
-export * from "./client-erc20";
-export * from "./client-addressList";
+export * from "./client-addressList/index";
+export * from "./client-erc20/index";
```

Comment:
`/index` is always inferred automatically. Unless there was a file called `addressList` in the parent folder, which would be an anti-pattern to avoid